README.md

```diff
@@ -1,3 +1,36 @@
+# Contribution Guide
+
+## Commit Code
+
+**deutsch**
+
+- Fork erstellen
+- Änderungen auf den fork commiten
+- Merge request machen
+- Pull Request reviewen lassen
+
+
+**english**
+
+- create a fork
+- commit changes to the fork
+- create a merge request
+- let the pull request be reviewed
+
```

Comment:
An alternative would be to link here one of the good guides like
- https://gist.github.com/Chaser324/ce0505fbed06b947d962
or
- https://help.github.com/articles/creating-a-pull-request-from-a-fork/